processing/data_collection/gazette/spiders/base.py

```diff
@@ -1,6 +1,10 @@
 # -*- coding: utf-8 -*-
 import dateparser
 import scrapy
+import re
+from datetime import datetime
+
+from gazette.items import Gazette
```

Comment:
These blocks are not following [PEP8 recommendations for imports](https://www.python.org/dev/peps/pep-0008/#imports), that is to say: one block for native modules, another one for third-party libraries and the third one for local modules:

```python
import re
from datetime import datetime

import dateparser
import scrapy

from gazette.items import Gazette
```